tests/transforms/test_classical_jacobian.py

```diff
@@ -133,9 +133,6 @@ def test_autograd_without_argnum(circuit, args, expected_jac, diff_method):
     if len(args) == 1:
```

Comment:
The note just above needs to be removed. (can't propose a change as it's outside of the commits range)